modules/appeals_api/config/shared_schemas/address.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "http://json-schema.org/draft-2020-12/schema#",
+  "description": "JSON Schema for VA Decision Review Forms: 'address'",
+  "$ref": "#/definitions/address",
+  "definitions": {
+    "address": {
+      "type": "object",
+      "properties": {
+        "addressLine1": { "type": "string", "minLength": 1, "maxLength": 60 },
+        "addressLine2": { "type": "string", "minLength": 1, "maxLength": 30 },
+        "addressLine3": { "type": "string", "minLength": 1, "maxLength": 10 },
+        "city":         { "type": "string", "minLength": 1, "maxLength": 60 },
+        "stateCode": {
+          "type": "string",
+          "enum": [
+            "AL", "AK", "AS", "AZ", "AR", "AA", "AE", "AP", "CA", "CO",
+            "CT", "DE", "DC", "FM", "FL", "GA", "GU", "HI", "ID", "IL",
+            "IN", "IA", "KS", "KY", "LA", "ME", "MH", "MD", "MA", "MI",
+            "MN", "MS", "MO", "MT", "NE", "NV", "NH", "NJ", "NM", "NY",
+            "NC", "ND", "MP", "OH", "OK", "OR", "PW", "PA", "PR", "RI",
+            "SC", "SD", "TN", "TX", "UT", "VT", "VI", "VA", "WA", "WV",
+            "WI", "WY"
+          ]
+        },
+        "countryCodeISO2": { "type": "string", "pattern": "^[A-Z]{2}$", "minLength": 2, "maxLength": 2 },
+        "zipCode5": {
+          "type": "string",
+          "description": "5-digit zipcode. Use \"00000\" if address is outside the United States",
```

Comment:
So how would this whole shared schemas thing work if we wanted to have an `address.json` that didn't require that `0's` be applied, but only two of our endpoints wanted to use it? I think we should consider how we want to version these shared schemas, since chances are we won't be able to stick to a single address definition indefinitely.

One idea was to just version the schema filename and hide that from our users with a default `schema_version` that gets passed in automatically by the controller to the `SchemaGetter`.

Reply:
Let me process this and then I may have questions or I'll jump on it.

Reply:
Done!